src/SprykerEco/Zed/UnzerApi/Business/Api/Request/Converter/CreateBasketRequestConverter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request\Converter;
+
+use ArrayObject;
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use SprykerEco\Zed\UnzerApi\Business\Api\Request\UnzerApiRequestConstants;
+
+class CreateBasketRequestConverter implements UnzerApiRequestConverterInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiRequestTransfer $unzerApiRequestTransfer
+     *
+     * @return array<string, string>
+     */
+    public function convertRequestTransferToArray(UnzerApiRequestTransfer $unzerApiRequestTransfer): array
+    {
+        $unzerApiCreateBasketRequestTransfer = $unzerApiRequestTransfer->getCreateBasketRequestOrFail();
+
+        return [
+            UnzerApiRequestConstants::PARAM_AMOUNT_TOTAL_GROSS => $unzerApiCreateBasketRequestTransfer->getAmountTotalGross(),
+            UnzerApiRequestConstants::PARAM_CURRENCY_CODE => $unzerApiCreateBasketRequestTransfer->getCurrencyCode(),
+            UnzerApiRequestConstants::PARAM_NOTE => $unzerApiCreateBasketRequestTransfer->getNote(),
+            UnzerApiRequestConstants::PARAM_ORDER_ID => $unzerApiCreateBasketRequestTransfer->getOrderId(),
+            UnzerApiRequestConstants::PARAM_BASKET_ITEMS => $this->convertBasketItems($unzerApiCreateBasketRequestTransfer->getBasketItems()),
+        ];
+    }
+
+    /**
+     * @param \ArrayObject|\Generated\Shared\Transfer\UnzerBasketItemTransfer[] $unzerBasketItemTransfers
+     *
+     * @return array<string, string>
```

Comment:
```suggestion
     * @return array<int, array<string, string>>
```

Or something like this